### DISCUSSION TITLE
IED System (ACE)?

### DISCUSSION THREAD

**Discussion by UnRealxInferno** [Vdauphin/HeartsAndMinds#1640] [Question and answer] [answered]
I'm playing with ACE loaded, but the IEDs that are placed still seem to be random objects like rubble, or dumpsters, rather than IEDS? how do I fix this?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#ied

Cheers

**Comment by UnRealxInferno** [comment]
Thanks, appears they're just really far into the floor, even when spotting difficulty is set to medium, so it's difficult to defuse, sometimes impossible, but from seeing other threads that seems intentional.
